scripts/katana_requirements/__main__.py

```diff
@@ -61,6 +62,12 @@ def setup_general_arguments(parser):
         default=[],
         action="append",
     )
+    parser.add_argument(
+        "--packaging-system",
+        "-p",
+        help="The packaging system to use. Only print packages which support this system.",
+        type=str,
```

Comment:
I don't suppose we can get a list of valid values here?

Reply:
The list is from the config file. I will add a reference to the `requirements packaging-systems` which does list the options.